docs_versioned_docs/version-ros2humble/components/changelog.css

```diff
@@ -0,0 +1,28 @@
+.changelog h2::before {
```

Comment:
Is there a reason why we have a copy of this file for each release rather than putting in somewhere in the /src directory? Maybe under /src or /static?

Reply:
My thinking was to keep the parallel structure of the different versions as much as possible, just in case there are additional changes we may need to make that don't affect all versions. Though I admit I struggle to think of a plausible example of such a change.